scripts/main/lychee.js

```diff
@@ -1173,3 +1173,80 @@ lychee.getBaseUrl = function () {
 		return location.href.replace(location.hash, "");
 	}
 };
+
+/**
+ * drag album to another one
+ * @param {DragEvent} ev
+ * @returns {void}
+ */
+lychee.startDrag = function (ev) {
+	/** @type string */
+	let type = "album";
+	/** @type ?Element */
+	let div = ev.target.closest("div.album");
+	if (!div) {
+		type = "photo";
+		div = ev.target.closest("div.photo");
+		if (!div) return;
+	}
+	ev.dataTransfer.setData("text", `${type}-${div.dataset.id}`);
```

Comment:
```suggestion
	/** @type ?HTMLDivElement */
	const div = ev.target.closest("div.album,div.photo");
	if (!div) return;
	const type = div.classList.contains("album") ? "album" : "photo";
	ev.dataTransfer.setData("text/plain", `${type}-${div.dataset.id}`);
```

 - More efficient, because it only traverses the DOM tree once and does not potentially traverse up to the root in case the first search for `div.album` fails
 - `const` is always better than `let` ;-)
 - According to the official HTML specs, the type must be a proper MIME type. Actually we could even use `application/json` and transfer a JSON object.